src/metabase/api/card.clj

```diff
@@ -105,7 +106,8 @@
                                                    [:like :c.dataset_query (format "%%#%s%%" model-id)]]]]
                         :where [:and [:= :m.id model-id] [:not :c.archived]]})
        ;; now check if model-id really occurs as a card ID
-       (filter (fn [card] (some #{model-id} (-> card :dataset_query query/collect-card-ids))))))
+       (filter (fn [card] (some #{model-id} (-> card :dataset_query query/collect-card-ids))))
+       (sort-by :name)))
```

Comment:
Some local tests suggest that the `sort-by` is not necessary, but looking at the query I have no idea how that could be, so I added it for reassurance. If you can prove that it's redundant I'm happy to remove it, though (and add an explanatory comment for bozos like myself)

Reply:
Perhaps incidental sorting in the order the database is returning them?

Since you're otherwise preserving the order of the query, why not use an `:order-by` instead?

Reply:
Yeah, I've done some more rigorous testing and there was a series of happy coincidences. `sort-by` definitely required; just transitioned it to `order-by` after some frustrating HoneySQL hiccups